tests/unit/commands/test_validation_sets.py

```diff
@@ -345,7 +345,9 @@ def test_edit_validation_sets_with_errors_not_amended(
     assert confirm_mock.mock_calls == [call("Do you wish to amend the validation set?")]
 
 
-def test_edit_yaml_error_retry(mocker, tmp_path):
+def test_edit_yaml_error_retry(mocker, tmp_path, monkeypatch):
+    monkeypatch.setenv("EDITOR", "vi")
```

Comment:
Good catch! Should we change this to a nonexistent command somewhere (perhaps in an auto-use fixture somewhere) to catch any time we make this assumption?

Reply:
will do